operations/services.md

```diff
@@ -408,12 +408,6 @@ Adds availability blocks which are used to group related [Availability update](#
 | --- | --- | --- | --- |
 | `AvailabilityBlocks` | array of [Availability block](#availability-block) | required | Availability blocks. |
 
-### Reservation purpose
-
-* Leisure
-* Business
-* Student
-
```

Comment:
This seems to be an incorrect change. The current flow is, that the dto is placed to the nearest usage. So this is the only correct place right now because the reservation purpose is used in availability block logic.